doc/Command-Reference.md

```diff
@@ -3106,19 +3106,19 @@ This command is the standard CMIS diagnostic control used for troubleshooting li
 
 - Usage:
   ```
-  sfputil debug loopback PORT_NAME LOOPBACK_MODE
+  sfputil debug loopback PORT_NAME LOOPBACK_MODE <enable/disable>
 
   Set the loopback mode
```

Comment:
```suggestion
  Valid values for LOOPBACK_MODE
```

Reply:
@mihirpat1  Thanks! It has been revised.